Sources/Commands/Workspace.swift

```diff
@@ -287,6 +323,39 @@ public class Workspace {
         try removeFileTree(dataPath)
     }
 
+    /// Find the managed dependencies for the given package name.
+    func findDependency(forURL url: String) throws -> ManagedDependency? {
```

Comment:
Is this worth a new function, it is just another way of spelling access to `dependencyMap`?